errors/errors.js

```diff
@@ -0,0 +1,17 @@
+exports.handlePsqlErrors = (err, req, res, next) => {
+    if (err.code) {
+        res.status(400).send({ msg: "Bad request" });
+    } else next(err);
+};
+
+exports.handleCustomErrors = (err, req, res, next) => {
+    if (err.msg) {
+        res.status(err.status).send({ msg: err.msg });
+    } else next(err);
+};
+
```

Comment:
We currently have no need for any PSQL or custom error handlers so we shouldn't include this yet.